cfme/markers/uncollect.py

```diff
@@ -52,8 +52,11 @@ def test_delete_all_snapshots(test_vm, provider_key, provider_type):
 
 """
 import inspect
+
 import pytest
 
+from cfme.utils.log import logger
+
 MARKDECORATOR_TYPE = type(pytest.mark.slip)
```

Comment:
i just noticed this one, lets please use a known mark name to avoid breaking strict